### PR TITLE
Add onto toolkit

### PR DESCRIPTION
**Pull request by bgruening**
This is replacing https://github.com/galaxyproject/tools-iuc/pull/6!

ping @peterjc @erasche

### CODE REVIEW

File: tools/onto_toolkit/get_child_terms.xml [hunk 1]
```diff
@@ -0,0 +1,48 @@
+<tool id="onto_tk_get_child_terms" name="Get the child terms" version="@VERSION@.0">
```
Comment (yhoogstrate) [inline tools/onto_toolkit/get_child_terms.xml]: haw about "Get child terms" ?

File: tools/onto_toolkit/get_parent_terms_by_relationship_type.xml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    <command detect_errors="aggressive">
+    perl '$__tool_directory__/get_parent_terms_by_relationship_type.pl'
+        '$input'
+        '$term_id'
+        '$rel_id' > '$output'
+    </command>
+    <inputs>
+        <expand macro="input_ontology" />
+        <expand macro="term_id" />
+        <param name="rel_id" type="text" value="" label="Rel ID" />
```
Comment (yhoogstrate) [inline tools/onto_toolkit/get_parent_terms_by_relationship_type.xml]: Relationship id?

**Comment by yhoogstrate** [comment]
would it make sense to add some validators to the params of `type="text"`?

File: tools/onto_toolkit/onto_macros.xml [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+            <yield />
+        </requirements>
+    </xml>
+    <xml name="input_ontology">
+        <param name="input" format="obo" type="data" label="Ontology file"/>
+    </xml>
+    <xml name="term_id">
+        <param name="term_id" type="text" value="" label="Term ID" />
+    </xml>
+    <xml name="citations">
```
Comment (yhoogstrate) [inline tools/onto_toolkit/onto_macros.xml]: I saw `<param name='rel_id'../>` popping by multiple times. should that be put into a macro too?

**Comment by bgruening** [comment]
@yhoogstrate thanks for the review! I hope I got everything.

**Pull request merged**